labs/RivePlayer/samples/RivePlayer.Samples/RivePlayerCustomSample.xaml

```diff
@@ -8,12 +8,25 @@
       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
       mc:Ignorable="d">
 
-    <StackPanel Spacing="15">
-        <TextBlock Text="1" />
-        <TextBlock Text="2" />
-        <TextBlock Text="3" />
-        <TextBlock Text="4" />
-        <TextBlock Text="5" />
-        <TextBlock Text="6" />
-    </StackPanel>
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="400" />
+        </Grid.ColumnDefinitions>
+        <labs:RivePlayer Width="600" Height="600" DrawInBackground="True" Source="https://public.rive.app/community/runtime-files/2244-4463-animated-login-screen.riv" Grid.Column="0">
+            <labs:BoolInput Target="isChecking" Value="{Binding IsChecked, ElementName=IsChecking}" />
```

Comment:
This `{Binding ` syntax used to work in an old UWP project, but the `OnValueChanged` event isn't firing now.